rm/rm-client/src/main/java/org/ow2/proactive/resourcemanager/common/event/RMInitialState.java

```diff
@@ -61,99 +57,87 @@
  * @since ProActive Scheduling 0.9
  */
 @PublicAPI
-@XmlRootElement
```

Comment:
What is the impact of removing this annotation?

Reply:
I will put it back